app/policies/vacancy_policy.rb

```diff
@@ -0,0 +1,13 @@
+class VacancyPolicy < ApplicationPolicy
+  def update?
+    record.owners.include?(user) || super
+  end
+
+  def create_with_group?(_group)
+    true
+  end
+
+  def replace_group?(_group)
+    true
```

Comment:
To be honest I copied this from the Article policy, since that has a similar permission structure, but I think it defines what relations you can add or update to the model. So we always allow someone (who is able to create or update the model) to update the group or add a group to the model. Because of course there are situations (not for this model) where you would only want a user to be able to add certain objects as a relation but not others, e.g. only objects that one is the owner of. But we simply allow the user to pick any group one wants.